src/components/home/HomeDesktop.jsx

```
@@ -1,21 +1,20 @@
-          alt="Código QR"
+          alt={getText("qr_code")}
           src="/images/qr-code-min.png"
           width="250"
           height="250"
         />
         <h1>TUS Santander</h1>
-        <span>
-          Escanea el código QR que se muestra en la pantalla usando la app{" "}
-          <b>Cámara</b> de tu móvil para acceder a la aplicación
-        </span>
+        <span dangerouslySetInnerHTML={{ __html: getText("desktop_instructions") }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)